lib/chef/knife/solo_prepare.rb

```diff
@@ -47,6 +47,10 @@ class SoloPrepare < Knife
           name, path = h.split("=")
           Chef::Config[:knife][:hints][name] = path ? JSON.parse(::File.read(path)) : Hash.new  }
 
+     option :lock_file,
```

Comment:
It appears the option is never used in `knife solo prepare`. I'm not sure you'd need it since I believe prepare only does on-node operations aside from writing the node JSON which should be different per host anyway.

Can you try removing it and see if it works? I'd rather make sure now than cause confusion down the road by merging it now.